### DISCUSSION TITLE
Thoughts about addition features

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#112] [Ideas]
In the past I’ve run a fairly complex multi-step event booking system, that served various not-profit organisations in the UK. Covid ended all the event. I’m now looking around at Hi.Events and I like basics of what it does, even though it doesn’t facilitate all that I would need. 

The key premise has been family groups booking for a festival style event, which includes children’s group activities and camping plots.

 The plots have a variety of sizes and types (tent, caravan or mobile home) and also include a paid option of electric hook up). 

An essential part of the process is that all the family is on one order, but life doesn’t always work for the family unit to book all at once, or things just change. I’ve needed in the past to be able to add or remove attendees to existing orders and to charge or refund any monetary difference. The system tracks what any change has been (ie X person removed, and shows the amount refunded and when, OR x person added and the additional amount charged, which is then shown on the order view).

This ability has been available for the event admin as well as the booking contact for their booking. Primary importance is for the admin team. 

I guess my question is whether Hi.Events could or would ever look to operate this way, or if there are suggestion for alternative ways of achieving the same?

**Comment by daveearley** [comment]
There are plans in the near future to support "add-ons," which should make situations like you described possible. Regarding order management, we will be working on enhancements to provide more flexibility in handling changes to existing bookings, including adding, removing, or changing attendees and adjusting charges or refunds accordingly.

For now, you should be able to achieve most of what you need using existing functionality. For plot sizes, you could create individual tickets. In the admin area, you can edit and refund attendees. While you can't currently charge an attendee when they're manually added, this ability will be added very soon.

**Reply by justynpride** [reply]
Thanks for the helpful information. I think the add-ons could solve a number of my scenarios going forward. they are outside a basic expectation for core booking system activities, but then essential for the specific event that needs them! 

I'm looking forward to seeing how this system develops. I've played with Attendize in the past, which I note you were involved with, but I think you have equaled and surpassed it even in these early stages!!

I note you are in Dublin - I've fond memories of it as my wife and I lived there for just over 4 years and had two boys while there.
